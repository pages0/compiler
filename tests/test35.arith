let x : <int> = ref 0 in
while !x<7 do x:= !x +1 end;
!x