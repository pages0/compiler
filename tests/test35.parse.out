( let [ x ( ref 0 ) ] ( ; ( while ( < ( ! x ) 7 ) ( != x ( + ( ! x ) 1 ) ) ) ( ! x ) ) )